website/static/js/projectorganizer.js

```diff
@@ -324,9 +324,6 @@ function _poToggleCheck(item) {
  */
 function _poResolveIcon(item) {
     var newIcon = Fangorn.Utils.resolveIconView(item);
-    if (newIcon === null) {
```

Comment:
With the removal of this code, `_poResolveIcon` is exactly equivalent to `Fangorn.Utils.resolveIconView`. Therefore, can we just set `resolveIcon` in Treebeard to `Fangorn.Utils.resolveIconView`?